xclim/testing/tests/test_indices.py

```diff
@@ -1082,8 +1094,8 @@ def test_jetstream_metric_woolings(self):
         np.testing.assert_equal(np.sum(~np.isnan(jetstr).data), 6)
         np.testing.assert_equal(jetlat.max().data, 19.0)
         np.testing.assert_equal(jetstr.max().data, 0.5620588628647811)
-        assert jetlat.units == "deg"
-        assert jetstr.units == "m s-1"
+        assert jetlat.units == da_ua.cf["latitude"].units
```

Comment:
currently: "deg" != "degrees_north"

Reply:
The problem was in the `declare_units` decorator! On exit, it converts the units to `pint` and then back to string, in order to ensure a CF-compliant output. However, `pint` thought "degrees_north" was simply an alias of "degrees". PR #959 fixes this by telling `pint` that it's a compatible but independent unit.